kensu/pyspark/spark_connector.py

```diff
@@ -603,237 +595,223 @@ def get_notebook_name():
+            provider_class_name = t2("spark_environment_provider", "io.kensu.sparkcollector.system.DefaultSparkEnvironmentProvider")
+
             ### How to get GIT info here?
             #         explicit_code_repo=None,
             #         explicit_code_version=None,
             #         explicit_code_maintainers=None,
             #         explicit_lauched_by_user=None,
-            code_repo_name = explicit_code_repo or os.environ.get("DAM_CODE_REPOSITORY", "Jupyter-Notebook:spark-" + spark.version+"::"+notebooks_path)
-            repository = t2("dam.activity.code.repository", code_repo_name)
+            code_repo_name = code_location or "Jupyter-Notebook:spark-" + spark_session.version + "::" + notebooks_path
+            repository = t2("code_location", code_repo_name)
             from datetime import datetime
             d = datetime.now()
 
-            code_version_value = explicit_code_version or os.environ.get("DAM_CODE_VERSION", notebook_name + "::" + str(d))
-            version = t2("dam.activity.code.version", code_version_value)
+            code_version_value = code_version or notebook_name + "::" + str(d)
+            version = t2("code_version", code_version_value)
+
+            user = t2("user_name", user_name)
 
-            global dam_version
-            dam_version = code_version_value
-            user = t2("dam.activity.user", explicit_lauched_by_user or dam_user_name)
-            organization = t2("dam.activity.organization", "Unknown")
             properties = jvm.scala.collection.mutable.HashSet()
 
             def add_prop(name, value):
                 if value is not None:
                     properties.add(t2(name, value))
 
-            properties.add(providerClassName)
-            properties.add(maintainers)
+            properties.add(provider_class_name)
             properties.add(repository)
             properties.add(version)
             properties.add(user)
-            properties.add(organization)
-            if ingestion_token:
-                properties.add(t2("dam.ingestion.auth.token", ingestion_token))
-            if is_offline is not None:
-                properties.add(t2("dam.ingestion.is_offline", is_offline))
-            if offline_mode_file_name:
-                properties.add(t2("dam.ingestion.offline.file", join_paths(dam_logs_directory, offline_mode_file_name)))
-            if allow_invalid_ssl_certificates is not None:
-                properties.add(t2("dam.ingestion.ignore.ssl.cert", allow_invalid_ssl_certificates))
+            if kensu_ingestion_token:
+                properties.add(t2("kensu_ingestion_token", kensu_ingestion_token))
+            if report_to_file is not None:
+                properties.add(t2("report_to_file", report_to_file))
+            if offline_file_name:
+                properties.add(t2("offline_report_dir_path", logs_dir_path))
+                properties.add(t2("offline_file_name", offline_file_name))
```

Comment:
also we probably want to sent `offline_report_dir_path` not depending if `offline_file_name` is set:
```suggestion
            if logs_dir_path:
                properties.add(t2("offline_report_dir_path", logs_dir_path))
            if offline_file_name:
                properties.add(t2("offline_file_name", offline_file_name))
```